app/models/ebean/Recipient.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2018 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.ebean;
+
+import com.google.common.base.MoreObjects;
+import io.ebean.Finder;
+import io.ebean.annotation.CreatedTimestamp;
+import io.ebean.annotation.UpdatedTimestamp;
+import models.internal.impl.DefaultEmailRecipient;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.PersistenceException;
+import javax.persistence.Table;
+
+/**
+ * Data Model for SQL storage of a report recipient.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ * @see Recipient.RecipientType
+ */
+// CHECKSTYLE.OFF: MemberNameCheck
+@Entity
+@Table(name = "recipients", schema = "portal")
+public final class Recipient {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "uuid")
+    private UUID uuid;
+
+    @CreatedTimestamp
+    @Column(name = "created_at")
+    private Timestamp createdAt;
+
+    @UpdatedTimestamp
+    @Column(name = "updated_at")
+    private Timestamp updatedAt;
+
+    @OneToMany(
```

Comment:
This should not be necessary and couples generic recipients to all consumptions of them (e.g. reports, alerts, etc.)